pyprof/prof/activation.py

```diff
@@ -51,37 +49,28 @@ def __init__(self, d):
         arg = args[0]
         assert (arg['type'] == "tensor")
 
-        self.i = arg
+        self.inp = Tensor(arg['shape'], arg['dtype'])
         self.dir = d.dir
 
     def params(self):
-        p = OrderedDict([('T', self.i['shape']), ('type', self.i['dtype'])])
-        return p
+        return str(self.inp)
 
     def flops(self):
-        direction = self.dir
-        tensor = self.i['shape']
-        t = self.i['dtype']
-
-        # TODO: revise
-        elems = Utility.numElems(tensor)
-        return elems
+        # TODO: revise based on op
+        return self.inp.size
 
     def bytes(self):
+        # TODO: revise based on op
         direction = self.dir
-        tensor = self.i['shape']
-        t = self.i['dtype']
-
-        elems = Utility.numElems(tensor)
-        elems = elems * (2 if direction == "fprop" else 3)
-
-        return elems * Utility.typeToBytes(t)
+        b = self.inp.bytes
+        b *= 2 if direction == "fprop" else 3
```

Comment:
Try to avoid magic numbers when possible. What is 2 and 3? Why those values? Named constants make this much more readable and maintainable.

Note I have seen these consts used in other files here. What if they change? You now have to change in many files. A single const, or better yet, a single shared `bytes` method would be much more robust.

Reply:
I assume `b` is for bytes, but try to avoid single letter variable names unless it is something common, like `i,j,k` for iteration and array indices.